docs/how-to-guides/set-global-configs.md

```diff
@@ -0,0 +1,58 @@
+# Set global configs
+Promptflow supports setting global configs to avoid passing the same parameters to each command. The global configs are stored in a yaml file, which is located at ~/.promptflow/pf.yaml by default.
+
+The config file is shared between promptflow extension and sdk/cli. Promptflow extension controls each config through UI, so the following sections will show how to set global configs using promptflow cli.
+
+## Set config
+```shell
+pf config set <config_name>=<config_value>
+```
+For example:
+```shell
+pf config set connection.provider="azureml:/subscriptions/<your-subscription>/resourceGroups/<your-resourcegroup>/providers/Microsoft.MachineLearningServices/workspaces/<your-workspace>"
+```
+
+## Show config
+The following command will get all configs and show them as json format:
+```shell
+pf config show
+```
+After running the above config set command, show command will return the following result:
+```json
+{
+  "connection": {
+    "provider": "azureml:/subscriptions/<your-subscription>/resourceGroups/<your-resourcegroup>/providers/Microsoft.MachineLearningServices/workspaces/<your-workspace>"
+  }
+}
+```
+
+## Supported configs
+### connection.provider
+The connection provider, default to "local". There are 3 possible provider values.
+#### local
+Set connection provider to local with `connection.provider=local`.
+
+Connections will be saved locally. `PFClient`(or `pf connection` commands) will [manage local connections](manage-connections.md). Consequently, the flow will be executed using these local connections.
+#### full azure machine learning workspace resource id
+Set connection provider to a specific workspace with:
+```
+connection.provider=azureml:/subscriptions/<your-subscription>/resourceGroups/<your-resourcegroup>/providers/Microsoft.MachineLearningServices/workspaces/<your-workspace>
+```
+
+When `get` or `list` connections, `PFClient`(or `pf connection` commands) will return workspace connections, and flow will be executed using these workspace connections.
+_Secrets for workspace connection will not be shown by those commands, which means you may see empty dict `{}` for custom connections._
+
+Note that `create`, `update` and `delete` commands are not supported for workspace connections, please manage it in workspace portal, az ml cli or AzureML SDK. 
```

Comment:
Can we get this line an adomnition tag?

Reply:
Feels like it's a little heavy, and if so, things about we can't show secrets should also be a note box?

Reply:
Added.